Modules/Options/TrackerTab/QuestieOptionsTracker.lua

```diff
@@ -53,6 +53,9 @@ function QuestieOptions.tabs.tracker:Initialize()
                         Questie.db.char.AutoUntrackedQuests = {}
                     end
                     QuestieTracker:Update()
```

Comment:
Do we need this call if we call with a timer? Also why do we need a timer in the first place?

There are multiple other timers below.

Reply:
`C_Timer.After(0.1, function() QuestieTracker:Update() end)`

For now, yes. Sadly. If ANY line in the tracker changes, then it's necessary. To put it simply, the first update nukes the Quest/Achievement & Quest Item button pools and clears all the 'set tags' then rebuilds the pools. Once that is complete the second update changes the formatting (width, height, etc.) and switches on or off any buttons that needs to be visible to the player based on each Quests/Achievements status. In the past I've tried separating this process with very little success. It creates FAR too many bugs and edge cases. In essence, nuking the entire tracker and rebuilding it from scratch seems to be the most efficient method. The QuestieTracker:Update() function is long and seems overly complicated but from an execution standpoint it's slim and fast. I've only been inserting `C_Timer.After(0.1, function() QuestieTracker:Update() end)` in places where its absolution needed. Even though it's called often, if the tracker is minimized and/or hidden. then most of it isn't even executed. The more a player tracks Quests and Achievements the more is executed. Make sense?

Reply:
Yes, this absolutely makes sense. I personally just get "a feeling" when I see timers in our code 😅 I am absolutely on your side that the code is really complex and it's dangerous to do major refactorings/code separations on it (... I failed in the past 🙈 )